accountsservice/io.elementary.pantheon.AccountsService.xml

```diff
@@ -1,16 +1,16 @@
 <node>
-  <interface name="io.elementary.greeter.AccountsService">
+  <interface name="io.elementary.pantheon.AccountsService">
```

Comment:
@danrabbit would `desktop` make more sense here? That seems to be the convention elsewhere for Pantheon stuff (`io.elementary.desktop`).

Reply:
From slack: Well, Pantheon is the name of the desktop, we can't change that at this point so either make sense to me but I'm more in favor of io.elementary.pantheon as it is a session-wide setting and not a "desktop-component" setting (which is a subtle nuance)